app/assets/stylesheets/_summary-card.scss

```diff
@@ -70,6 +79,21 @@
     margin-left: 0;
   }
 
+  .app-summary-card__tag {
+    @include govuk-font($size: 19);
+    width: 33%;
+    margin: 0;
+    padding: 0;
+    display: block;
+    white-space: nowrap;
+
+    @include govuk-media-query($from: desktop) {
+      display: inline-block;
+      margin-left: govuk-spacing(2);
+      text-align: right;
+    }
+  }
```

Comment:
I guess you've rebased from my branch here? So I won't merge until the stylesheet changes are in

Reply:
Stylesheet changes have been merged so this PR can be progressed

Reply:
👍